libs/metrics/src/lib.rs

```diff
@@ -45,6 +47,14 @@ macro_rules! register_uint_gauge {
     }};
 }
 
+#[macro_export]
+macro_rules! register_f64_gauge_vec {
+    ($NAME:expr, $HELP:expr, $LABELS_NAMES:expr $(,)?) => {{
+        let gauge_vec = F64GaugeVec::new($crate::opts!($NAME, $HELP), $LABELS_NAMES).unwrap();
```

Comment:
Unsure why we need a gauge, are we planning on updating the value? a `metrics::register_counter_vec!` could work (already exported)?

Reply:
I would usually only use a counter for something where the delta between samples is meaningful -- I think of the semantic meaning as "one could call `rate()` on this".

Reply:
Oki, makes sense. How is this different from the `prometheus::GaugeVec` which is already re-exported before these changes?

Reply:
ah - it didn't jump out at me that the unqualified GaugeVec happened to be f64.  Added a commit that removes this redundant specialization